README.md

```diff
@@ -83,7 +79,7 @@ Ubuntu 14.04     | 5.18
 Ubuntu 16.04     | 5.22
 
 Zonemaster has been tested with the combination of OS and Perl version listed in the table
-above. Other versions of Perl might work. Perl versions earlier than 5.14.2 will _NOT_ work.
+above. Perl versions earlier than 5.14.2 are not supported.
```

Comment:
This changes the meaning. Perl versions earlier than 5.14.2 will not work. Perl version 5.22 has been tested to work with Ubuntu 16.04, but not with Ubuntu 14.04, but it might work with Ubuntu 14.04.

Reply:
@matsduf What do your think of this wording? "Perl versions earlier than 5.14.2 are explicitly not supported." That clearly implies that there's a gray area that may or may not work.

Reply:
@mattias-p, I think that wording is harder to interpret than the current text. What is the problem with the current text?

Reply:
A theme that will be recurring in all of my edits is that I want to see specificity. Phrasings like "might work" are imprecise and don't convey anything of substance to the user. In this case, and if a user wants to give a different combination of OS/Perl a shot, there isn't really anything we can do to help them if we haven't tested and confirmed it ourselves. They can choose to try it or not, and a line from us about how they can do exactly that doesn't really provide any actual assistance.

Reply:
We signal that it is worth trying other versions of Perl, and especially other OS/Perl version combinations, but nothing lower than Perl 5.14.2.

Reply:
@JohanIIS and @mattias-p, I suggest:

Zonemaster requieres Perl version 5.14.2 or higher. Zonemaster has been tested with the default version of Perl in the OSs as listed in the table above.